core/snippets/snippet_types.py

```diff
@@ -30,3 +30,16 @@ def get_variable_strict(self, name: str):
         if variable is None:
             raise ValueError(f"Snippet '{self.name}' has no variable '{name}'")
         return variable
+
+
+@dataclass
+class InsertionSnippet:
+    body: str
+    scopes: list[str] = None
+
+
+@dataclass
+class WrapperSnippet:
+    body: str
+    variableName: str
```

Comment:
@pokey In your example code in the previous poll request this was a list. Was that just a typo?

Reply:
Yep